pkg/apis/kops/util/labels.go

```diff
@@ -22,18 +22,21 @@ import (
 	"k8s.io/api/core/v1"
 )
 
-func GetNodeRole(node *v1.Node) string {
-	role := ""
+func GetNodeRoles(node *v1.Node) string {
```

Comment:
Should return `[]string` I think.  Although because you are going to test it for membership repeatedly, you could also return a `sets.String` (which does need to be initialized, with `sets.NewString()`)